irksome/pc.py

```diff
@@ -54,16 +49,11 @@ def getAtilde(self, A):
     def form(self, pc, test, trial):
         """Implements the interface for AuxiliaryOperatorPC."""
         appctx = self.get_appctx(pc)
-        butcher = appctx["butcher_tableau"]
-        F = appctx["F"]
-        t = appctx["t"]
-        dt = appctx["dt"]
-        u0 = appctx["u0"]
-        bcs = appctx["bcs"]
-        stage_type = appctx.get("stage_type", None)
-        bc_type = appctx.get("bc_type", None)
-        splitting = appctx.get("splitting", None)
-        vandermonde = appctx.get("vandermonde", None)
+        stepper = appctx["stepper"]
+        butcher = stepper.butcher_tableau
+        F = stepper.F
+        u0 = stepper.u0
+        bcs = stepper.orig_bcs
```

Comment:
As above, now we're pulling things indirectly out of `stepper`, not directly from the `appctx`, so we should reconsider what we're putting in there.

Reply:
Agreed.  Do we need actually need anything besides the stepper in the app context?

Reply:
We currently stash `splitting` and `bc_type` in `StageCoupledTimeStepper`, and some similar bits elsewhere.  But it looks to me like all of these are also stashed in `stepper` directly.  So, I'd say we go the simple route and just stash `stepper`...